src/fstar/FStar.Interactive.fs

```diff
@@ -384,31 +389,66 @@ let update_deps (filename:string) (m:modul_t) (stk:stack_t) (env:env_t) (ts:m_ti
 let rec go (line_col:(int*int))
            (filename:string)
            (stack:stack_t) (curmod:modul_t) (env:env_t) (ts:m_timestamps) : unit = begin
+  let (dsenv: DsEnv.env), (tcenv: TcEnv.env) = env in
   match shift_chunk () with
-  | Info(symbol, fqn_only, pos_opt) ->
+  | Info(symbol, pos_opt) ->
     let dsenv, tcenv = env in
     let info_at_pos_opt = match pos_opt with
       | None -> None
-      | Some (file, row, col) -> FStar.TypeChecker.Err.info_at_pos (snd env) file row col in
+      | Some (file, row, col) -> FStar.TypeChecker.Err.info_at_pos tcenv file row col in
     let info_opt = match info_at_pos_opt with
       | Some _ -> info_at_pos_opt
       | None -> // Use name lookup as a fallback
-        if symbol = "" then None
+        if symbol = "" || DsEnv.try_current_module dsenv = None then None
```

Comment:
When we get an info query before getting a `#push` with a `module` declaration. Will add a comment.